app/src/main/java/app/revanced/integrations/sponsorblock/SponsorBlockUtils.java

```diff
@@ -403,6 +403,8 @@ public static void notifyShareBtnVisibilityChanged(View v) {
     }
 
     public static String appendTimeWithoutSegments(String totalTime) {
+        if (SettingsEnum.shorts_playing) return "";
```

Comment:
Should this not be:
```suggestion
        if (SettingsEnum.shorts_playing) return totalTime;
```
?